tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -19,13 +19,49 @@ module.exports = {
         .click('@coreMenuOpenButton')
         .waitForElementVisible('@coreMenu')
         .waitForAnimationToFinish()
+    },
+    clickOnMarkAsRead: function () {
+      return this.waitForElementVisible('@ocTopbar')
+        .initAjaxCounters()
+        .waitForOutstandingAjaxCalls()
```

Comment:
I don't think you need to wait here for Ajax calls, or is it to wait for the 30sec till its queried again?

Reply:
I tried so... to be honest: I don't know what these two methods do but as like 98% of acceptance tests use them I did so as well

Reply:
this fuctions are useful e.g. if you click some button that triggers some AJAX actions. We want to wait till all of those are finished before doing the next thing.
But they will also introduce some artificial waiting, but if we are waiting for something else we should not miss-use them as `pause()`